specification/network/resource-manager/Microsoft.Network/stable/2019-12-01/virtualWan.json

```diff
@@ -3553,7 +3553,7 @@
           }
         },
         "bgpSettings": {
-          "$ref": "./virtualNetworkGateway.json#/definitions/BgpSettings",
+          "$ref": "#/definitions/VpnGatewayBgpSettings",
```

Comment:
This might be a breaking change in SDKs

Reply:
We have decided to define a new VpnBgpSettings for vpngateway since we are not using BgpPeeringAddress property of BgpSettings . 
@nalinrajgupta Can you please help to answer this question if I missed anything ?
@number213 Do you have any suggestion for bringing this change without any impact ?

Reply:
@number213 : We just removed BgpPeeringAddress property as it was always null !

Reply:
Wel, the issue is that it will be a breaking change for customers that use strongly-types SDKs (e.g. Net SDK)